packages/core/src/App/Containers/UrlUnavailableModal/url-unavailable-modal.scss

```diff
@@ -0,0 +1,40 @@
+.dc-modal__container_url-unavailable-modal {
+    .dc-modal {
+        &-header__title,
+        &-footer {
+            padding: 2.4rem;
+        }
+        &-body {
+            padding: 0 2.4rem;
+
+            ul {
+                margin: 1rem 0;
+
+                li:before {
+                    content: '•';
```

Comment:
Sorry, is it for the same purpose as `list-style-type` for `ul`?

Reply:
no, `list-style-type: disc` doesn't give me the correct dot appearance.